sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,113 @@
+# SIG Docs 2020 Annual Report
+
+**Authors:**
+
+Irvi Aini ([@irvifa](https://github.com/irvifa)),
+Jim Angel ([@jimangel](https://github.com/jimangel)),
+Karen Bradshaw ([kbhawkey](https://github.com/kbhawkey)),
+Kaitlyn Barnard ([@kbarnard10](https://github.com/kbarnard10)),
+Taylor Dolezal ([onlydole](https://github.com/onlydole)),
+Tim Bannister ([sftim](https://github.com/sftim))
+
+- [Operational](#operational)
+- [Membership](#membership)
+- [Current initiatives and project health](#current-initiatives-and-project-health)
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md](/committee-steering/governance/sig-governance.md)?**
+
+  - **Is your README accurate? have a CONTRIBUTING.md file?**
+
+    Yes. We also have `Contributing.md` file.
+  - **All subprojects correctly mapped and listed in [sigs.yaml](/sig-list.md)?**
+
+    Yes.
+      
+  - **What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date?
+    Are you keeping recordings up to date/trends in community members watching recordings?**
+    
+    Quite small (usually 5-10 people) and generally we try to make people feel comfortable to give their opinion.
+    Meeting notes are up to date. We have recording of the meetings uploaded.
+
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired?
+Are OWNERS files up to date in these areas?**
+
+Yes it is.
+
+**How does the group get updates, reports, or feedback from Working Groups? Are there any springing up or being retired? 
+Are OWNERS files up to date in these areas?**
+
+Yes it is.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+<!-- TODO -->
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes, they are.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+PR are generally open for anyone to review. We have [PR wrangler rotation](https://github.com/kubernetes/website/wiki/PR-Wranglers).
+In general each of the approver will taking the rotation. Under exceptional circumstances where there's lots of
```

Comment:
nit: I think line 60 (and the included link) explains the PR rotation. I would rewrite line 61 as:

Each approver participates in the PR wrangler rotation. Approvers are supported during their rotation by other reviewers and approvers.